.gitignore

```diff
@@ -23,7 +23,6 @@ checkstyle.iml
 checkstyle.ipr
 checkstyle.iws
 .idea
-
```

Comment:
Remove this change

Reply:
done

Reply:
this file should not have any changes , please do

Reply:
Should I remove it and keep the line empty or delete the line?

Reply:
revert all changes from this file.
keep it untouched

Reply:
done